plugins/chat/db/post_migrate/20221104054957_backfill_channel_slugs.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class BackfillChannelSlugs < ActiveRecord::Migration[7.0]
+  def up
+    channels = DB.query(<<~SQL)
+      SELECT chat_channels.id, COALESCE(chat_channels.name, categories.name) AS title, NULL as slug
+      FROM chat_channels
+      INNER JOIN categories ON categories.id = chat_channels.chatable_id
+      WHERE chat_channels.chatable_type = 'Category'
```

Comment:
It is post migration, so there is 0.00001%chance that slug will be not null, so we need to add `AND chat_channels.slug is NULL` ?

Reply:
Yes good catch!